code/modules/research/experimentor.dm

```diff
@@ -500,6 +491,7 @@
 		if(globalMalf > 16 && globalMalf < 35)
 			visible_message("<span class='warning'>[src] melts [exp_on], ian-izing the air around it!</span>")
 			throwSmoke(loc)
+			var/mob/trackedIan = locate(/mob/living/simple_animal/pet/dog/corgi/Ian) in GLOB.mob_living_list
```

Comment:
```suggestion
			var/mob/tracked_ian = locate(/mob/living/simple_animal/pet/dog/corgi/Ian) in GLOB.mob_living_list
```
while you're editing this code, do you mind converting this var names to `snake_case`

Reply:
Done!